Terminal.Gui/Input/Command.cs

```diff
@@ -4,10 +4,35 @@
 
 namespace Terminal.Gui {
 
+
+	/// <summary>
+	/// Defines the scope of a <see cref="Command"/> that has been bound to a key with <see cref="View.AddKeyBinding"/>.
+	/// </summary>
+	public enum CommandScope {
+		/// <summary>
+		/// The command is scoped to just the view that adds the key binding.
+		/// </summary>
+		View = 0,
+
+		/// <summary>
+		/// The command is scoped to the view that adds the key binding and that view's <see cref="View.SuperView"/>.
+		/// </summary>
+		SuperView,
```

Comment:
Should this be called `CommandScope` or `KeyBindingScope`?

If you can define two different keys for the same Command with different scopes (e.g. below).  Then the scoping is of the keybinding not the command itself?

```csharp
AddKeyBinding (Key.F9, CommandScope.SuperView, Command.ToggleExpandCollapse);
AddKeyBinding (Key.Esc, CommandScope.View, Command.ToggleExpandCollapse);
```